apps/fz_http/lib/fz_http_web/live/mfa_live/register_steps_component.ex

```diff
@@ -70,6 +70,10 @@ defmodule FzHttpWeb.MFA.RegisterStepsComponent do
 
       <div class="has-text-centered">
         <canvas data-qrdata={@uri} id="register-totp" phx-hook="RenderQR" />
+
+        <div class="mb-4">
+          <a href="#" id="copy-totp-uri" phx-hook="ClipboardCopy" data-clipboard={@uri}>Copy URI</a>
```

Comment:
Sorry for not clarifying — I think we probably want this to a be “Can’t scan the QR code? Click here to show the secret key” which displays a `<pre><code>` block containing the secret when clicked (that should make it easy for the user to either type it in manually or highlight and copy-paste).

I *think* this key should be the same data stored in the QR code. If you added it to Google Authenticator for example or another password manager it should accept it and start generating valid codes.

Reply:
👌 Will get back to this tomorrow.

Reply:
👍🏼

Reply:
Please check the updated screenshots. I don't feel like I need to hide it.